### DISCUSSION TITLE
move users, settings, world and so on from minecraft server to itzg/docker-minecraft-server?

### DISCUSSION THREAD

**Discussion by DariBer** [itzg/docker-minecraft-server#1930] [Q&A] [answered]
I have a Minecraft Java Server (v.1.19.3) running on a Linux (Ubuntu), no extra mods or stuff installed.
I'm trying to find any info if it's possible to move all the users, settings, map/world and so on from that server to the itzg/docker-minecraft-server

**Comment by itzg** [comment, marked as answer]
Yes, you can. Assuming the use of docker compose, which I highly recommend, do the following:

1. Create a directory
2. Put a `docker-compose.yml` in there with the content from [this example](https://github.com/itzg/docker-minecraft-server/blob/master/examples/docker-compose-simple.yml)
3. Create a subdirectory called `data`
4. Put everything from the old server into that `data` directory

**Reply by DariBer** [reply]
Everything is up and running now, but I have a question now, when Minecraft release a new update, when using the folder data, will it also update the minecraft_server.1.19.3.jar file aswell and rename it and so on?

Also will it update the JDK when needed?

Thank you for great job

**Reply by itzg** [reply]
> Everything is up and running now, but I have a question now, when Minecraft release a new update, when using the folder data, will it also update the minecraft_server.1.19.3.jar file aswell and rename it and so on?

Yes, it will. Do keep in mind that  it can only check during startup. So, when there is an update you'll just need to restart the container.  

> Also will it update the JDK when needed?

Assuming you're always using the default or `latest` image tag, then yes. 

> Thank you for great job

You're welcome 😀
